docs/docs/10.4-test-utils.md

```diff
@@ -16,15 +16,22 @@ Simulate.{eventName}(DOMElement element, object eventData)
 
 Simulate an event dispatch on a DOM node with optional `eventData` event data. **This is possibly the single most useful utility in `ReactTestUtils`.**
 
-Example usage:
+**Clicking an element**  
+```javascript
+var node = React.findDOMNode(this.refs.button);
+React.addons.TestUtils.Simulate.click(node);
+```
 
+**Changing the value of an input field and then pressing ENTER**  
 ```javascript
 var node = React.findDOMNode(this.refs.input);
-React.addons.TestUtils.Simulate.click(node);
-React.addons.TestUtils.Simulate.change(node, {target: {value: 'Hello, world'}});
-React.addons.TestUtils.Simulate.keyDown(node, {key: "Enter"});
+node.value = 'giraffe'
```

Comment:
I think this is only needed if you're reading directly from the node anywhere (like in event handler or after), as opposed to reading from `event.target.value` (which is what the previous example code was assuming).

I generally think that doing `expect(something about dom node)` isn't really the default case and we should instead mention that as an aside. I'd rather we kept what we had and then call out separately that Simulate does not change the DOM node, so if your event handler or other code reads directly from the DOM you need to make that change separately.